test/functional/test-activity.js

```diff
@@ -116,7 +118,8 @@ describe('Activity getTotalEngagedTime', () => {
       scrollObservable.add(handler);
     });
 
-    fromClassForDoc(ampdoc, 'activity', Activity);
+    markElementScheduledForTesting(fakeWin, 'amp-analytics');
```

Comment:
No, but admittedly it looks weird. See similar code on master [here](https://github.com/kmh287/amphtml/blob/master/test/functional/test-url-replacements.js#L73)